app/controllers/reviews_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ReviewsController < ApplicationController
   before_filter :authenticate_user!
-
+  RATINGS_OPTIONS = [1, 2, 3, 4, 5].freeze
```

Comment:
not quite what I was thinking. Could you set this constant in the `Review` model itself.